workflows/cluster_plots/__main__.py

```diff
@@ -0,0 +1,60 @@
+import json
+import os
+
+import luigi
+from google.cloud import storage
+
+from birdclef.knn_labels import (
+    compute_embedding_2d,
+    get_knn_labels,
+    get_label_agreement,
+    get_subset_pdf,
+    plot_distances,
+    plot_embedding,
+    write_plots_to_disk,
+)
+from birdclef.utils import get_spark
+from workflows.utils.pull import Pull
+
+from .plotting import ClusterPlotAllTasks
+
+
+class ClusterPlotTaskWrapper(luigi.WrapperTask):
+    output_path = luigi.Parameter()
+    list_species = luigi.Parameter()
+
+    def requires(self):
+        pull_data = Pull(
+            input_path="data/processed/birdclef-2022/birdnet-embeddings-with-neighbors",
+            output_path="../data/processed/birdclef-2022/birdnet-embeddings-with-neighbors",
```

Comment:
We want to run this script from the root of the repo like `python -m workflows.cluster_plots`. This should be a parameter anyways that's passed in via the luigi.build function.